template.rb

```diff
@@ -151,15 +152,15 @@ def app_dir
   end
 
   gsub_file(f, "HELLOWORLD") do |_match|
-    app_name.underscore.capitalize
+    app_name.underscore.upcase
   end
 
   gsub_file(f, "hello_world") do |_match|
     app_name.underscore
   end
 
   gsub_file(f, "helloworld") do |_match|
-    app_name.underscore
+    url_safe_name
```

Comment:
Have run the the template to generate a new project to make sure evey thing still works?

Reply:
Yep, it all should be working fine. Tests all passed and I got it to run and everything